source/image/ImageBuffer.cpp

```diff
@@ -194,10 +194,15 @@ bool ImageBuffer::Read(const ImageFileData &data, int frame)
 
 
 namespace {
+	void ReadPNGInput(png_structp png_ptr, png_bytep outBytes, png_size_t byteCountToRead)
+	{
+		static_cast<iostream *>(png_get_io_ptr(png_ptr))->read(reinterpret_cast<char *>(outBytes), byteCountToRead);
```

Comment:
camelCase
```suggestion
	void ReadPNGInput(png_structp pngStruct, png_bytep outBytes, png_size_t byteCountToRead)
	{
		static_cast<iostream *>(png_get_io_ptr(pngStruct))->read(reinterpret_cast<char *>(outBytes), byteCountToRead);
```

Reply:
I'm pretty sure this is just copied from somewhere else in this file, but sure.